### PR TITLE
Pass image & version as arguments

### LINKED ISSUE #32: Makefile to create operator registry image should take the operator image as argument
See this comment: https://github.com/redhat-developer/devopsconsole-operator/pull/29#discussion_r267269894

### PR DESCRIPTION
**Pull request by baijum**
Fixes #32

### CODE REVIEW

File: test/olm/Dockerfile.registry [hunk 1]
```diff
@@ -1,7 +1,9 @@
 FROM quay.io/openshift/origin-operator-registry:latest
 
-# The operator image location should be updated before copying.
+ARG image
```
Comment (kwk) [inline test/olm/Dockerfile.registry]: What about having a default?

Comment (baijum) [reply test/olm/Dockerfile.registry]: This `Dockerfile` is called from a `Makefile` target.  There is a default value set in that `Makefile`. The default value is `quay.io/redhat-developers/devopsconsole-operator`.

Comment (kwk) [reply test/olm/Dockerfile.registry]: Where exactly is it being passed to `docker build` in the Makefile?

Comment (baijum) [reply test/olm/Dockerfile.registry]: The code still not merged. It is here: https://github.com/redhat-developer/devconsole-operator/pull/33/files#diff-57dae1b5b313e109d7ebdfd77a500c5cR154

Comment (kwk) [reply test/olm/Dockerfile.registry]: But why did you then merge this PR before the other one?

Comment (baijum) [reply test/olm/Dockerfile.registry]: It was possible to create the image manually. I have described the steps in the original PR description. The manual steps are given here: https://github.com/redhat-developer/devconsole-operator/tree/master/test#steps-to-verify-operator-registry

**Pull request merged**